professional-services/roles/head-of-engineering.md

```diff
@@ -0,0 +1,67 @@
+# Head of Engineering
+
+The Head of Engineering (HoE) oversees the engineering team that creates solutions within the Professional Services team. They are ultimately responsible for the output from the engineering team, and support them in their business as usual functions. They are the arbiters of engineering standards and interfacing with the rest of the business.
```

Comment:
The Head of Engineering (HoE) oversees the engineering team in the delivery of bespoke, Professional Services solutions.